lib/ExtUtils/MANIFEST.SKIP

```diff
@@ -38,6 +39,11 @@
 # and Module::Build::Tiny generated files
 \b_build_params$
 
+^local/   # Carton local library
```

Comment:
I'm slightly confused what a carton local library would be doing in a CPAN dist.

Reply:
(Come to think of it, the `cpanfile.snapshot` should probably be excluded too.)